index.js

```diff
@@ -11,15 +11,16 @@ const pagination = document.querySelector('[data-js="pagination"]');
 
 // States
 const maxPage = 1;
-const page = 1;
+let page = 1;
 const searchQuery = "";
 
 async function fetchCharacters() {
   cardContainer.innerHTML = "";
   const response = await fetch(
-    "https://rickandmortyapi.com/api/character/?page=1"
+    `https://rickandmortyapi.com/api/character/?page=${page}`
   );
   const data = await response.json();
+  console.log("data: ", data);
```

Comment:
```suggestion
  console.log("data: ", data);
```

Remove Console Logs in production code :)